### ISSUE TITLE
Back button not visible in example app on iOS

### ISSUE THREAD

**Issue by draggie** [openmobilehub/react-native-omh-storage#74]
In the example app the back button to go to previous folder is the same color as the background color causing it to be invisible, it can be fixed easily with adding

`headerTintColor: theme.colors.text`

like 

```
 const screenOptions = useMemo(
    () => ({
      contentStyle: {
        backgroundColor: theme.colors.background,
      },
      headerStyle: {
        backgroundColor: theme.colors.primary,
      },
      headerTintColor: theme.colors.text,
    }),
    [theme.colors]
  );
```

in the `RootNavigationContainer.tsx`

**Comment by adamTrz** [comment]
Thanks for finding that @draggie 
Guess it should be fixed with #68 

Could you mind taking a look please? :)

**Comment by draggie** [comment]
Yes, just checked, it works fine now
